controllers/survey.controller.js

```diff
@@ -15,15 +18,86 @@ async function getSurveyData(user_id) {
     });
 }
 
+
+const getdata = (request, response) => {
+    let count = 0;
+    pool.query('SELECT * from survey', (error, results) => {
+        console.log("result--->", results.rows);
+        let data = results.rows
+        data.forEach(item => {
```

Comment:
@sonalratho26 
Add checks before using array functions in this case(line number 27) if `data` is undefined or the length of the `data` is 0 then the code will break at run time. 
Use following everytime you use array methods on array. `data && data.length > 0 && data.forEach(() => {})`

Please correct this.

Reply:
I have correct this sir